pyro/infer/traceenum_elbo.py

```diff
@@ -13,23 +13,35 @@
 from pyro.util import check_model_guide_match, check_site_shape, check_traceenum_requirements, torch_isnan
 
 
+def _dict_iadd(dict_, key, value):
+    if key in dict_:
+        dict_[key] = dict_[key] + value
+    else:
+        dict_[key] = value
```

Comment:
nit: this can be better written as: `dict_[key] = dict_.get(key, 0) + value`

Reply:
That would add an extra tensor op `0 + tensor`.